integration_test/staking_module/staking_test.yaml

```diff
@@ -0,0 +1,28 @@
+- name: Staking a validator
```

Comment:
nit: We can use a more meaningful test names like Test Staking a validator should xxx